frontend/js/src/user-feed/UserFeed.tsx

```diff
@@ -622,145 +518,119 @@ export default class UserFeedPage extends React.Component<
+          ) : (
+            <>
+              <div
+                id="timeline"
+                data-testid="timeline"
+                style={{ opacity: isLoading ? "0.4" : "1" }}
               >
-                <a
-                  role="button"
-                  onClick={this.handleClickOlder}
-                  onKeyDown={(e) => {
-                    if (e.key === "Enter") this.handleClickOlder();
-                  }}
-                  tabIndex={0}
-                  href={!nextEventTs ? undefined : `?max_ts=${nextEventTs}`}
+                <ul>
+                  {events?.map((event) => {
+                    const { created, event_type, user_name } = event;
+                    return (
+                      <li
+                        className="timeline-event"
+                        key={`event-${user_name}-${created}`}
+                      >
+                        <div className="event-description">
+                          <span className={`event-icon ${event_type}`}>
+                            <span className="fa-layers">
+                              <FontAwesomeIcon
+                                icon={faCircle as IconProp}
+                                transform="grow-8"
+                              />
+                              <FontAwesomeIcon
+                                icon={getEventTypeIcon(event_type) as IconProp}
+                                inverse
+                                transform="shrink-4"
+                              />
+                            </span>
+                          </span>
+                          {renderEventText(event)}
+
+                          <span className="event-time">
+                            {preciseTimestamp(created * 1000)}
+                            {renderEventActionButton(event)}
+                          </span>
+                        </div>
+
+                        {renderEventContent(event)}
+                      </li>
+                    );
+                  })}
+                </ul>
+              </div>
+              <div
+                className="text-center"
+                style={{
+                  width: "50%",
+                  marginLeft: "auto",
+                  marginRight: "auto",
+                }}
+              >
+                <button
+                  type="button"
+                  className="btn btn-primary btn-block"
+                  onClick={() => fetchNextPage()}
+                  disabled={!hasNextPage || isFetchingNextPage}
                 >
-                  Older &rarr;
-                </a>
-              </li>
-            </ul>
-          </div>
-          <div className="col-md-offset-1 col-md-4">
-            <UserSocialNetwork user={currentUser} />
-          </div>
+                  {(isLoading || isFetchingNextPage) && "Loading more..."}
+                  {!isFetchingNextPage &&
+                    (hasNextPage ? "Load More" : "Nothing more to load")}
```

Comment:
Just realised, the message should not be displayed if it is loading
```suggestion
                  {!(isLoading || isFetchingNextPage) &&
```